WordPress/Classes/Extensions/MediaLibrary+Local.swift

```diff
@@ -143,7 +168,7 @@ extension MediaLibrary {
                         filesToKeep.insert(localThumbnailURL.lastPathComponent)
                     }
                 }
-                try purgeLocalDirectory(exceptFiles: filesToKeep)
+                try purgeLocalDirectory(exceptFiles: filesToKeep, type: .uploads)
```

Comment:
Yep! Haven't implemented the different cleanup methods yet, based on the the directory types.